src/Wallabag/CoreBundle/Helper/EntriesExport.php

```diff
@@ -23,17 +25,25 @@ class EntriesExport
     private $entries = [];
     private $author = 'wallabag';
     private $language = '';
+    private $user;
 
     /**
      * @param TranslatorInterface $translator  Translator service
      * @param string              $wallabagUrl Wallabag instance url
      * @param string              $logoPath    Path to the logo FROM THE BUNDLE SCOPE
      */
-    public function __construct(TranslatorInterface $translator, $wallabagUrl, $logoPath)
+    public function __construct(TranslatorInterface $translator, $wallabagUrl, $logoPath, TokenStorageInterface $tokenStorage)
     {
```

Comment:
As php-cs-fixer says, comment here should be updated here to reflect the new method argument.